09LessonNine/in-class-project/index.html

```diff
@@ -6,5 +6,16 @@
 </head>
```

Comment:
include a css reset or at least set the body's margin to 0 so that there's no scrolling on the page